src/node_crypto.cc

```diff
@@ -4357,8 +4477,15 @@ void Verify::VerifyFinal(const FunctionCallbackInfo<Value>& args) {
     hbuf = Buffer::Data(args[1]);
   }
 
+  int padding = RSA_PKCS1_PADDING;
+  int saltlen = -2;
+  if (args.Length() >= 4 && args[3]->IsObject()) {
```

Comment:
Fwiw you don’t need the `args.Length()` part of the check, if `args[3]` is out of bounds it will just be `undefined`. I don’t mind keeping it if you think that helps with readability, though.